### PR TITLE
Add concurrency

### PR DESCRIPTION
**Pull request by bigtiger**
🍏 

- [x] Uses `express-cluster` to take advantage of multiple cores.

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -42,12 +45,15 @@ const listen = (port) => {
...
 const port = process.env.PORT || 3002;
-if (port > 3002) {
-  listen(port);
-} else {
-  findPort(port, port + 100, function(ports) {
-    return listen(ports[0]);
-  });
-}
+
+cluster(function() {
```
Comment (rylnd) [inline src/server.js]: Does the function given to `cluster` need to instantiate the app? [Their documentation](https://www.npmjs.com/package/express-cluster#synopsis) seems to indicate so.

Comment (bigtiger) [inline src/server.js]: RealTalk™ I don't have any idea what I'm doing.

But! To answer your question I would say, yes, that sounds correct, and I believe that is happening on lines `51` or `54` in the updated version.

**Comment by rylnd** [comment]
@bigtiger when I run try to run this I get a looping error, presumably from the `cluster` call failing to fork: 

```
[server] SyntaxError: Unexpected token import
[server]     at createScript (vm.js:53:10)
[server]     at Object.runInThisContext (vm.js:95:10)
[server]     at Module._compile (module.js:543:28)
[server]     at Object.Module._extensions..js (module.js:580:10)
[server]     at Module.load (module.js:488:32)
[server]     at tryModuleLoad (module.js:447:12)
[server]     at Function.Module._load (module.js:439:3)
[server]     at Module.runMain (module.js:605:10)
[server]     at run (bootstrap_node.js:427:7)
[server]     at startup (bootstrap_node.js:151:9)
[server] /Users/ryland/code/work/adorable/avatars-api/index.dev.js:1
[server] (function (exports, require, module, __filename, __dirname) { import 'babel-polyfill';
[server]                                                               ^^^^^^
```
ad infinitum.

**Comment by rylnd** [comment]
> @bigtiger when I run try to run this I get a looping error, presumably from the cluster call failing to fork

If you don't see that, it might be that my usage of nvm does not persist across forks and I'm running an old version of node for those processes; that would explain the error. Let me try something quick...

**Comment by bigtiger** [comment]
No, we see that behavior in dev. :eyes:ing

**Comment by bigtiger** [comment]
@rylnd Fixed with @itsthatguy's help.

**Comment by rylnd** [comment]
It doesn't look like I can approve this PR now (weird), but 👍

**Pull request merged**